recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -457,30 +449,18 @@ CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_VIDEO_TI_CAL=m
 CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_VIDEO_TI_CAL=m
 CONFIG_VIDEO_OV2659=m
 CONFIG_VIDEO_OV5640=m
-# CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 CONFIG_DRM_I2C_NXP_TDA998X=y
-CONFIG_DRM_PANEL_SIMPLE=y
-CONFIG_DRM_PANEL_OSD_OSD101T2587_53TS=y
```

Comment:
Same remark for this: correctly dropped, we don't have this. Did you check manually?